### ISSUE TITLE
build issues with node 0.6

### ISSUE THREAD

**Issue by SheetJSDev** [ljharb/actions#1]

**Comment by ljharb** [comment]
Yes, node 0.6 hasn't built in Github Actions for awhile - it broke a long time ago when GHA changed its container type. I've left it because it's optional and because I still want to figure out how to fix it.
